README.md

```diff
@@ -47,7 +46,7 @@ envs:
 This task overrides the path to the global `datadog-ci.config.json` file.
 
 ```yml
-- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
+- datadog-mobile-app-upload@1: {}
```

Comment:
Having an empty object only makes sense when you don't follow the YAML key with properties. Here, you have  `inputs` just below, so it's not needed.

```suggestion
- datadog-mobile-app-upload@1:
```

Reply:
good point 👍